src/main/java/pl/javastart/task/PersonBuilder.java

```diff
@@ -0,0 +1,45 @@
+package pl.javastart.task;
+
+import java.util.Scanner;
+
+public class PersonBuilder {
+    private Scanner scanner = new Scanner(System.in);
+
+    int i = 1;
+
+    private Person createPerson() {
+        System.out.print("Podaj imię: ");
+        String firstname = scanner.nextLine();
+        System.out.print("Podaj wiek: ");
+        int age = scanner.nextInt();
+        scanner.nextLine();
+        return new Person(firstname, age);
+    }
+
+    Person[] createPersonArray(int size) {
+        Person[] persons = new Person[size];
+        Person firstPerson = createPerson();
+        for (int i = 0; i < persons.length; i++) {
+            persons[i] = firstPerson;
+        }
+        Person personToAdd = createPerson();
+        checkArrayDuplicate(persons, personToAdd);
+        personToAdd = createPerson();
+        checkArrayDuplicate(persons, personToAdd);
```

Comment:
Mocno skomplikowane to rozwiązanie, ciężko powiedzieć dlaczego metoda createPerson jest wywoływana akurat 3 razy i czy na pewno np. dla tablicy o rozmiarze 10 nadal będzie to działało.
Algorytm w tym zadaniu to:
//1. Utwórz nowy obiekt
//2. Sprawdź unikalność
//3. jeżeli obiekt jest unikalny, to wstaw na kolejne wolne miejsce

W filmie pokazuję jak do tego podejść lepiej